src/lib.rs

```diff
@@ -340,6 +273,41 @@
 //! See ["Error handling"](#error-handling) for more details on [`handle_error`]
 //! and error handling in general.
 //!
+//! ## Wildcard routes
+//!
+//! axum also supports wildcard routes:
+//!
+//! ```rust,no_run
+//! use axum::{
+//!     handler::get,
+//!     Router,
+//! };
+//!
+//! let app = Router::new()
+//!     // this matches any request that starts with `/api`
+//!     .route("/api/*rest", get(|| async { /* ... */ }));
```

Comment:
Since also "/api/bar" matches this route, I think it would be easier to understand to leave "rest" out of "/api/*rest"

Reply:
What do you mean? There is no `bar` in this route? And wildcard captures must be named so `/api/*` is not valid.

Reply:
If wildcard captures must be named, then my comment wrong.
Thanks for answering!